Remotion/Core/UnitTests/Reflection/TypeDiscovery/AssemblyFinding/AssemblyFinderIntegrationTest.cs

```diff
@@ -30,6 +30,7 @@ namespace Remotion.UnitTests.Reflection.TypeDiscovery.AssemblyFinding
 {
   [TestFixture]
   [Serializable]
+  [Ignore ("TODO RM-7808: Integrate the RoslynCodeDomProvider and renable the AssemblyCompiler tests")]
```

Comment:
Please add a preprocessor `!NETFRAMEWORK` to those ignores since the code works in net framework.